packages/sdk/python/src/relayfile/writeback_consumer.py

```
@@ -0,0 +1,84 @@
+            if self._stopped:
+                return
+            handler = next(
+                (candidate for candidate in self._handlers if candidate.can_handle(item.path)),
+                None,
+            )
+            if handler is None:
+                self._ack_failure(item, Exception(f"No writeback handler found for path: {item.path}"))
+                continue
+            try:
+                handler.execute(item, self._provider)
+            except Exception as exc:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)